### ISSUE TITLE
Way to load test data (w/ ground truth results) from a file.

### ISSUE THREAD

**Issue by damellis** [damellis/ESP#240]
In some cases, we can get good data sets online. It might be useful to have a way to load those data sets (w/ their ground truth results) into the ESP interface so we can see evaluate our algorithms against them. This is particularly true in the signal processing case, where the reference data is just test data -- as opposed to the machine learning case, where the reference data could be used as training data (in which case we might want to do k-folds validation testing instead, as in #116). 

This also requires an evaluation function for comparing the ground truth data to the results of the signal processing / machine learning pipeline. There may be some standard metric(s) we can use, but an example author may also want to supply their own.

**Comment by damellis** [comment]
This is a duplicate of #144, #145, and #146. I added more comments to those and am closing this.
